TeX_files/chapter03.tex

```diff
@@ -0,0 +1,227 @@
+%\usepackage{enumitem,amssymb} - pakiet odpowiedzialny za kwadraciki do zakreślania
+%\usepackage{multicol} - pakiet odpowiedzialny za podział tekstu na kolumny
```

Comment:
Hej :) Byłoby idealnie, gdyby deklaracje paczek znalazły się w pliku main.tex :)

Reply:
Hej :) Spoko, w następnym commicie tak zrobię.

Reply:
Hej :) Spoko, tak będę robił już teraz. W sumie to nie ustaliliśmy w ramach kolektywu kiedy wstawiamy formatowanie. Możnaby to przegadać.

PS.Sorry za zamieszanie z kontami, mam drugie konto na GitHubie (takie założone kiedyś tam w przeszłości) i jeszcze nie ogarniam do końca jak się przełączać między nimi.

Reply:
Tak, myślę, że to wszystko się jeszcze poukłada w międzyczasie :) 

Najważniejsze tylko, żeby nie wyrzucało błędów podczas budowania PDF. Czy u Ciebie to działa dobrze? 

W
`%\usepackage{enumitem,amssymb} - pakiet odpowiedzialny za kwadraciki do zakreślania`
% - mówi, że cała linia  jest komentarzem, więc, żeby zadeklarować paczkę powinno być:
`\usepackage{enumitem,amssymb} %- pakiet odpowiedzialny za kwadraciki do zakreślania`

Reply:
Commit z poprawkami dodany. Powinno być ok.